### ISSUE TITLE
Make this work for our guild

### ISSUE THREAD

**Issue by P47CK** [pelekon/RcLootCouncil-Wotlk#9]
Hello, 
Our guild on Frosthold - Sunwell tested this addon, and the features it has is very intriguing. We would really like to use this addon for the rest of this server, and for upcoming Wotlk servers. However, it seems to not work properly for us.

Current issues we noticed:
*Re-announces loot every time ML loots the boss/mob
*Every time you make a change in settings, everyone who has the addon have to relog to save settings
*Can't see history of loot received/rewarded
*Can't see what's active on filter of votes
*Would be awesome to have ELVUI skin

@ajseward I am also writing this because it seems you have disabled the "Issues" tab on your forked version, but as said, it would be awesome to have the addon if it worked properly! We would LOVE to use this addon SO MUCH!

Anyways, here is my discord info: p47rick#3303

**Comment by andrew6180** [comment]
My version is only related to this one in the fact it was originally just planned to fix most of the issues on this one.


The one on my fork is a full new backport of RCLootCouncil 2.0 and we've been using it with minimal issues in our guild for ~1 month now. 

https://github.com/ajseward/RcLootCouncil-Wotlk/releases/tag/2.0.4

Honestly did not know that issues were disabled but I've enabled them now.

**Comment by pelekon** [comment]
Hey, to be honest I dropped hit addon and I don't plan to ever get back to it as I stopped playing on wotlk and private servers overall. Furthermore,  I saw Ajseward's repo and his version is far more developed.

TLDR: closing issue as project is dropped by me.
